common/logisticspipes/proxy/specialinventoryhandler/StorageDrawersInventoryHandler.java

```diff
@@ -155,36 +179,60 @@ public int roomForItem(ItemIdentifier item) {
 	@Override
 	public int roomForItem(ItemIdentifier itemIdent, int count) {
 		int room = 0;
-		for(int i=0; i < _drawer.getDrawerCount(); i++) {
+		for (int i = 0; i < _drawer.getDrawerCount(); i++) {
+			if (!_drawer.isDrawerEnabled(i))
+				continue;
+
 			IDrawer drawer = _drawer.getDrawer(i);
-			if(drawer != null && drawer.getStoredItemCopy() == null) {
-				count += drawer.getMaxCapacity();
-			} else if(drawer != null && drawer.canItemBeStored(itemIdent.makeNormalStack(1))) {
-				count += drawer.getRemainingCapacity();
+			if (drawer == null)
+				continue;
+
+			ItemStack protoStack = itemIdent.makeNormalStack(1);
+			if (drawer.canItemBeStored(protoStack)) {
+				if (drawer.isEmpty()) {
+					room += drawer.getMaxCapacity(protoStack);
+				}
+				else {
+					room += drawer.getRemainingCapacity();
+				}
 			}
 		}
-		if(count != 0) {
-			return Math.min(room, count);
-		} else {
-			return room;
-		}
+
+		return room;
```

Comment:
Probably because it was added at a later point. Ignore it for now (don't add the count != 0). I will probably remove the count at some point.